WordPress/src/main/java/org/wordpress/android/viewmodel/mlp/ModalLayoutPickerViewModel.kt

```diff
@@ -46,62 +48,74 @@ class ModalLayoutPickerViewModel @Inject constructor(
     val buttonsUiState: LiveData<ButtonsUiState> = _buttonsUiState
 
     /**
-     * Tracks the Modal Layout Picker list items
+     * Tracks the layout categories
      */
-    private val _listItems = MutableLiveData<List<ModalLayoutPickerListItem>>()
-    val listItems: LiveData<List<ModalLayoutPickerListItem>> = _listItems
+    private val _layoutCategories = MutableLiveData<List<LayoutCategoryUiState>>()
+    val layoutCategories: LiveData<List<LayoutCategoryUiState>> = _layoutCategories
+
+    /**
+     * Tracks the categories
+     */
+    private val _categories = MutableLiveData<List<CategoryListItemUiState>>()
+    val categories: LiveData<List<CategoryListItemUiState>> = _categories
```

Comment:
Wondering if we can use a single UiState object that represents the state of the screen 🤔.  

Internal ref: paqN3M-1W

[Example](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/ui/reader/discover/interests/ReaderInterestsViewModel.kt#L196)

Reply:
I agree with you Ashita. That will make things a lot clearer. I tackled this with 5897942a501cf0793ce0395afc1c073fe3371a31
Note that I plan to enhance and use the loading/error states in the next PR that will implement the service calls.

Reply:
Fixed in 5897942